solution/src/main/openapi/solution.yaml

```diff
@@ -177,43 +184,66 @@ paths:
         required: true
         schema:
           type: string
-    post:
-      operationId: addOrReplaceParameters
+    patch:
+      operationId: updateSolutionParameters
       tags:
         - solution
-      summary: Add Parameters. Any item with the same ID will be overwritten
+      summary: Update solution parameters
       requestBody:
-        description: the Parameters
+        description: Parameters to update
         required: true
         content:
           application/json:
             schema:
               type: array
               items:
                 $ref: "#/components/schemas/RunTemplateParameter"
+            examples:
+              Parameters:
+                value:
+                  - id: "param1"
+                    labels:
+                      en: "Parameter 1"
+                      fr: "Paramètre 1"
+                    varType: "string"
+                    defaultValue: "default"
```

Comment:
This example and the YAML one just below should probably be factorized and `$ref`ed.
There are others below too: `ParameterGroups`, `RunTemplates` and `RunTemplate` at least.